src/engine/sidechain/engineshoutcast.cpp

```diff
@@ -198,6 +200,10 @@ void EngineShoutcast::updateFromPreferences() {
     m_ogg_dynamic_update = (bool)m_pConfig->getValueString(
             ConfigKey(SHOUTCAST_PREF_KEY,"ogg_dynamicupdate")).toInt();
 
+    // Dynamic Ogg metadata update
```

Comment:
That I would also know.. Even code is very trivial and if I understood correct changing Ogg stream metadata was problem in some libshout and if you don't tap that checkbox metadata won't be updated.